src/main/kotlin/racingcar/Driver.kt

```diff
@@ -0,0 +1,11 @@
+package racingcar
+
+class Driver(private val car: Car) {
+    val driveRecord = mutableListOf<Int>()
+    fun drive(isMovable: Boolean) {
```

Comment:
어떤 곳은 프로퍼티와 메서드의 간격이 떨어져있는데 이곳은 아니네요 일관성있게 맞춰주는게 좋을것 같아요!
```suggestion
    val driveRecord = mutableListOf<Int>()
    
    fun drive(isMovable: Boolean) {
```

Reply:
넵넵...!